internal/pkg/service/stream/storage/test/file.go

```diff
@@ -39,20 +41,22 @@ func NewFileOpenedAt(openedAtStr string) model.File {
 	fileKey := NewFileKeyOpenedAt(openedAtStr)
 	return model.File{
 		FileKey: fileKey,
-		Type:    model.FileTypeCSV,
```

Comment:
We completely get rid of Type, so it is not possible to define that we store anything else than CSV, slice as CSV etc?

Reply:
Yes but I would expect that `file.Type` == `csv`. It makes quite a sense

Reply:
> Yes but I would expect that `file.Type` == `csv`. It makes quite a sense

Yes, but the encoding pkg is responsible for that, and in order to connect it somehow sensibly, that's how it is now:
- `File.LocalStorage.Encoding.Type`
- `Slice.LocalStorage.Encoding.Type`